packages/atomic/src/components/result-template-components/atomic-result-text/atomic-result-text.tsx

```diff
@@ -60,23 +78,23 @@ export class AtomicResultText {
         ></atomic-component-error>
       );
     }
-    const value = ResultTemplatesHelpers.getResultProperty(
-      this.result,
-      this.field
-    );
 
-    if (value === null) {
+    if (!this.resultValue && !this.default) {
```

Comment:
How about setting the default for the prop to be `No title` (localized). 

And we never remove a result.

Because then you get things like "Results 1-10 of XXX", but you actually see 9 results. 

It brings a whole another suite of problem/wtf moment to realize that "you are seeing only 9 results because one of them has no title".

Reply:
That only remove the result text component, not it's parent result. Sometimes the field might not be filled and we want to remove the component, so it doesn't cause styling issues, with flex for example